content/docs/1.6.0/deploy/important-notes/index.md

```diff
@@ -23,6 +23,20 @@ in an old version of longhorn-instance-manager until it is detached (by scaling
 reattached (by scaling its consuming workload up). Consider scaling workloads down and back up again as soon as possible
 after upgrading from a version without this mechanism (v1.5.1 or older) to v{{< current-version >}}.
 
+### Default Priority Class
+
+Longhorn v1.6.0 introduces the default Priority Class `longhorn-critical`, which has the highest value and ensures that Longhorn pods are not evicted by kube-scheduler when system resources are low.
+
+During upgrades, Longhorn applies the default Priority Class to components depending on specific settings.
+
+- When all volumes are detached and the Priority Class default setting is not customized, the default Priority Class is applied to all Longhorn components. The default Priority Class setting `priority-class` is updated.
+- When all volumes are detached and you have specified a value for the Priority Class default setting, the default Priority Class is applied only to user-deployed components. The default Priority Class setting `priority-class` is not updated.
```

Comment:
> The default Priority Class setting `priority-class` is not updated.

The meaning of sentence 1 has now changed so I need to clarify the meaning of sentence 2. Do you mean this:
"The default Priority Class setting `priority-class` **of system-managed components** is not updated."

Reply:
> The default Priority Class setting priority-class is not updated

That's a global setting `priority-class` https://longhorn.io/docs/1.5.3/references/settings/#priority-class and it will apply to all components if it is changed.

Maybe it should be 
> The global setting `priority-class` is not updated